charts/app-config/image-tags/production/govuk-sli-collector

```diff
@@ -0,0 +1,3 @@
+image_tag: latest
+automatic_deploys_enabled: false
+promote_deployment: false
```

Comment:
Better omit these two since they won't work (as you correctly identified already).
```suggestion
```